main.py

```diff
@@ -66,7 +66,17 @@ def main():
                 #cast the string back into an int
                 student_id = int (student_id)
 
-                grade = input("Enter Grade: ")
+
+                isGradeEmpty = True
+                while (isGradeEmpty):
+                    grade = input("Enter Grade: ") 
+
+                    if (not grade or grade == "" or grade.startswith(" ")):
```

Comment:
It should be something like:
`if (not grade or grade == "" or not grade.strip())`

But it's okay for this lab. The lab's purpose is mainly know you are good with GitHub flow, and merges.